rholang/src/main/k/rholang/free.k

```diff
@@ -137,6 +137,123 @@ rule <thread> ...
          <NodeCounter> (NEWID:Int => NEWID +Int 2) </NodeCounter>
     ... </FreeVars> ... </thread>
 
+
+
+// Logical "and"
+// First for a conjunctin of HigherProcesses
+rule <thread> ...
+        <FreeVars> ...
+          <node> [ID:Int][ P1:HigherProc /\ P2:HigherProc ][ PARENT:Exp ][ (.List
+            => ListItem(NEWID) ListItem(NEWID +Int 1)) ]{ FREEVARS:List }[ (false
+            => true) ] </node>
+         (.Bag => <node> [NEWID][ P1 ][ ID ][ .List ]{ .List }[false]  </node>
+                  <node> [NEWID +Int 1][ P2 ][ ID ][ .List ]{ .List }[false]  </node>)
+         <NodeCounter> (NEWID:Int => NEWID +Int 2) </NodeCounter>
+    ... </FreeVars> ... </thread>
+// Now for a conjuction of Chans
+rule <thread> ...
```

Comment:
I'm trying to figure out where a logical connective between channels would be used.  I guess it would look something like
```
match P {
  for (_ <- y /\ @{Q|R}) { _ } => Nil
}
```
But we could get the same effect with

```
match P {
  for (_ <- @{y /\ {Q|R}} { _ } => Nil
}
```
except that `y` would be a pattern variable instead of a name variable, so uses of it in name position would need to be quoted.

So I don't think you need to support logical connectives between channels.

Reply:
Do we say that it's incorrect syntax to write a conjunction of channels then?

Reply:
(Took it out.)